android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java

```diff
@@ -342,7 +374,13 @@ public void setProgress(int max, int progress, boolean indeterminate)
 	public Notification buildNotification()
 	{
 		Notification notification = notificationBuilder.build();
-		notification.flags = this.flags;
+		
+		if (hasProperty(TiC.PROPERTY_GROUP_KEY)) {
+			// remove FLAG_AUTO_CANCEL as this will prevent group notifications
+			this.flags &= ~Notification.FLAG_AUTO_CANCEL;
+		}
+		notification.flags |= this.flags;
```

Comment:
Never mind. The "Notification.Builder" sets flags (such as group summary) that we don't want to clear with an assignment.